README.md

```diff
@@ -58,6 +61,6 @@ The CLI will use `solid` as the initialiation keyword. The CLI commands will the
   - `add`: Starts the process of submitting your current project to our ecosystem listing (Solidex) ie. `solid ecosystem publish`
   - `search`: Initializes an ecosystem search result `solid ecosystem search auth`
 
-# Contributions
+## Contributing
 
 Please feel free to contribute to this repo by expanding on this design document. Once we lock a general design a choice of technology will be decided.
```

Comment:
Now that work is being done instead of just designing the implementation, we should update this? Maybe a `CONTRIBUTING.md`?

Reply:
Sounds like a good idea. I've just made an issue for it: https://github.com/solidjs-community/solid-cli/issues/31